### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,24 +22,25 @@ env:
...
+    - php: 7.4
+      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1 PHPUNIT_COVERAGE_TEST=1 MAINTENANCE=2.x-dev
 
 before_script:
   # Init PHP
   - phpenv rehash
   - phpenv config-rm xdebug.ini || true
 
   - composer require --no-update silverstripe/recipe-cms "$RECIPE_VERSION"
+  - composer require --no-update sminnee/phpunit-mock-objects:^3
```
Comment (robbieaverill) [inline .travis.yml:42]: Would you mind commenting why this is here so future devs know, and can decide when it's safe to remove?

Comment (emteknetnz) [reply .travis.yml:42]: have added comment

File: .travis.yml [hunk 2]
```diff
@@ -9,24 +22,26 @@ env:
...
     - php: 7.1
-      env: DB=PGSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1 MAINTENANCE=^2
     - php: 7.2
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_COVERAGE_TEST=1 MAINTENANCE=2.x-dev
+      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
+    - php: 7.3
+      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
+    - php: 7.4
+      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1 PHPUNIT_COVERAGE_TEST=1 MAINTENANCE=2.x-dev
```
Comment (robbieaverill) [inline .travis.yml]: Same here, you don’t need to do both

Comment (emteknetnz) [reply .travis.yml]: updated

**Review by robbieaverill — APPROVED** [review]
Merge on green

**Pull request merged**